### PR TITLE
Project lead

### PR DESCRIPTION
**Pull request by yongtang**
This change is considered as part of the GOVERNANCE.md update (11/12/2018)
to add the project lead in OWNERS file. The term (11/11/2019) aligns
with GOVERNANCE.md update time (11/12/2018).

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Comment by fturib** [comment]
+1

**Comment by yongtang** [comment]
+1

**Comment by chrisohaver** [comment]
+1

**Comment by rajansandeep** [comment]
+1

**Review by bradbeam — APPROVED** [review]

**Review by chrisohaver — APPROVED** [review]

**Review by johnbelamaric — APPROVED** [review]

**Comment by yongtang** [comment]
6 lgtm so far, needs several more

**Review by jameshartig — APPROVED** [review]

**Review by pmoroney — APPROVED** [review]

**Comment by fturib** [comment]
Please do not merge, even though we have the majority. 
The OWMER file is processed automatically, we need to update that process to allow comments after the github handle in the approvers section.

**Comment by yongtang** [comment]
Let's also leave the PR for a day or two, so that everyone have a chance to express their opinions.

Our governance text was designed to allow everyone to have a chance to voice any opinion or concerns (in weeks).

This PR is considered as part of the governance text update (11/12/18), for the time constraints now.

Though it is good to leave a reasonable time frame for everyone to have a chance to involve if possible.

**Comment by yongtang** [comment]
The PR is open, feedbacks from maintainers and any community members welcome.

**Review by greenpau — APPROVED** [review]

**Review by grobie — APPROVED** [review]

**Review by miekg — APPROVED** [review]

**Comment by miekg** [comment]
hard part is fixing the damn yaml to parse this cleanly :/

**Comment by dilyevsky** [comment]
Should we just make it a `<user>*`?

**Review by Isolus — APPROVED** [review]

**Comment by miekg** [comment]
@yongtang just make it comment. See https://github.com/coredns/coredns/pull/2304

Seems easiest and doesn't require rewriting everything.

**Comment by yongtang** [comment]
@miekg Updated PR. Will that work?

**Comment by miekg** [comment]
yes, super super nit: drop the `-` in front of the data. Or has that some kind of significance?

**Pull request merged**
